### PR TITLE
Feature/load assembly

### PR DESCRIPTION
**Pull request by thibauthourlier**
# Requirements
When creating your Pull request, please fill out the template below:

# PR details
_Is this a fix/ update/ new feature?_
New feature
_Include a short description_
New sub pipeline config for loading the assembly
_Include links to JIRA tickets_
https://www.ebi.ac.uk/panda/jira/browse/ENSGENEBUI-469
# Testing
_Have you tested it?_
Yes
# Assign to the weekly GitHub reviewer
_If you are a member of Ensembl, please check the Genebuild weekly Rotas and assign this week's GitHub reviewer to the PR_
I am the reviewer for this sprint

### CODE REVIEW

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    production_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
```
Comment (ens-carlos) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm:167]:
```suggestion
      -dbname => 'ensembl_production',
```
Should there be a variable for the "dbname" in the same way as for the other db details?

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+      -driver => $self->o('hive_driver'),
+    },
+
+    taxonomy_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ncbi_taxonomy',
```
Comment (ens-carlos) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm:176]:
```suggestion
      -dbname => 'ncbi_taxonomy',
```

Should there be a variable for the "dbname" in the same way as for the other db details?

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
+      -driver => $self->o('hive_driver'),
+    },
+
+    taxonomy_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
```
Comment (ens-carlos) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm:173]:
```suggestion
      -host   => $self->o('production_db_server'),
      -port   => $self->o('production_db_port'),
```
Should there be different variables for taxonomy_db_server and port?

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm [hunk 4]
```diff
@@ -0,0 +1,557 @@
...
+      -parameters => {
+        'target_db'        => $self->o('reference_db'),
+        'output_path'      => $self->o('output_path'),
+        'enscode_root_dir' => $self->o('enscode_root_dir'),
+        'production_db'    => $self->o('production_db'),
+      },
+      -rc_name    => 'default',
+      -flow_into  => {
+        1 => WHEN ('#load_toplevel_only# == 1' => ['process_assembly_info'],
+            '#load_toplevel_only# == 2' => ['custom_load_toplevel'],
```
Comment (ens-carlos) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm:234]:
```suggestion
            '#load_toplevel_only# == 2' => ['custom_load_toplevel'],
```
It's fine, maybe a perltidy indentation issue? but not sure if easy to configure.

**Review by ens-carlos — APPROVED** [review]
Looks good!

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm (hunk 4 above)
Comment (thibauthourlier) [reply modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm:234]: I didn't use perltidy, I added indentation but maybe not how you were expected. I see the suggestion as being the same code as before

**Review by ens-carlos — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**